_rules/document-has-headings-for-main-section-of-content-047fe0.md

```diff
@@ -28,11 +28,11 @@ This rule applies to any [HTML web page][].
 
 ## Expectations
 
-The first non-[decorative][] node in the [flat tree][] which is inside the [main block of content][] of the test target and has a non-empty [accessible name][]:
+For each test target, all the following are true for the first non-[decorative][] node in it which is inside the [main block of content][] of this test target and has a non-empty [accessible name][]:
```

Comment:
This means exactly the same as it was before. Is this changed purely because the sentence needs to begin with "for each test target..." ? I am fine with the new sentence, just making sure I understand it.

Reply:
Yes. @WilcoFiers wanted both the "for each" and the "all the following are true" bits…
https://github.com/act-rules/act-rules.github.io/pull/421#discussion_r524127304
https://github.com/act-rules/act-rules.github.io/pull/421#discussion_r524149442